package.json

```diff
@@ -2,6 +2,7 @@
   "name": "preflight-test-project-react-passing",
   "version": "0.1.0",
   "private": true,
+  "packageManager": "pnpm@8.3.1",
```

Comment:
what is this? can you link to the docs associated with this? is this a standard configuration option in `package.json`?

I think there is a more standard value for this - called "engines" or something. if we can use this, let's use this instead

Reply:
this may be relevant for all of the projects we want to deploy on Netlify during the course - so you may need to also update the instructions in lectures / the Command Line cheatsheets

Reply:
I used this because I found out that Netlify uses `pnpm version 7.x` as default. Here is the [Link](https://docs.netlify.com/configure-builds/available-software-at-build-time/#tools)

In the documentation for Netlify, it says that I can specify a pnpm version when editing the package.json file and add `"packageManager": "pnpm@8.3.0"`. Here is the [Link](https://docs.netlify.com/configure-builds/manage-dependencies/#pnpm)

I will research, maybe there is another solution for this

Reply:
oh`"packageManager"` is from Corepack / Node.js:

https://nodejs.org/api/corepack.html#configuring-a-package

Reply:
but can you use a tag like `latest` instead of a locked version number which we will have to maintain?

Reply:
I tried to use `packageManager: pnpm@latest`, but Netlify throws an error while deploying 
```
Found pnpm version (Usage Error: Invalid package manager specification in package.json; expected a semver version
```
When testing this locally, I also received an error message:
```
String does not match the pattern of "(npm|pnpm|yarn)@\d+\.\d+\.\d+(-.+)?".
```

I didn't find any other solution so far, so maybe we have to maintain this

Reply:
ok since you're doing it in the ESLint config `bin/install.js`, then let's just maintain the version number over there, until [Netlify upgrades to pnpm v8+](https://answers.netlify.com/t/upgrade-netlify-build-to-pnpm-v8/90570)